tests/x11regressions/libreoffice/libreoffice_mainmenu_components.pm

```diff
@@ -0,0 +1,130 @@
+# LibreOffice tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "x11test";
+use strict;
+use testapi;
+
+# Case 1503827 - LibreOffice: Launch application components from system menu
+
+# open desktop mainmenu and click office
+sub open_mainmenu() {
+    my $self = shift;
+
+    wait_still_screen;
+    send_key "alt-f1";
+    assert_screen 'test-desktop_mainmenu-1', 20;
```

Comment:
Why the 20 second assert_screens on this and other lines? Is there something wrong with the default timeout (which is 30 seconds)?

Good Practice is to leave the timeout alone unless there is a very good reason why you must match it faster.. and I do not see this in this case

*NOTE* This isn't critical and I might still merge the test anyway, but I'd much rather it be perfect and follow good practices